### ISSUE TITLE
Instructions for building from scratch

### ISSUE THREAD

**Issue by Lerbytech** [reymond-group/tmap#19]
Last summer I really needed tmap working on my pc, but package didn't work due to "unspecified cpu instruction".
I had ubuntu 18.06. My friend had this distro working on his mac, and another friend on Arch.. well he didn't wanted to build it)
Also there was no package for Windows. So... i had to build it from scratch.

Tmap assembly instructions

You need to know that:

- Main url - https://github.com/reymond-group/tmap
- The article is here - https://arxiv.org/abs/1908.10410
- The site is here (go through the web archive) - http://tmap.gdb.tools/
- Tmap depends on the _ogdf_ package. The repository contains ogdf and tmap. Although the ogdf can be downloaded pre-built from conda repository, I built both - since the error was in the ogdf part.

OGDF related links:
- https://ogdf.uos.de/doc/
- https://github.com/ogdf/ogdf
- https://ogdf.uos.de/releases/

Also there is anaconda repository - https://anaconda.org/tmap/tmap

All packages were collected by the author through this script - https://github.com/reymond-group/tmap/blob/master/azure-pipelines.yml

The test method of the assembled package is to run the script https://github.com/reymond-group/tmap/blob/master/tmap/tests/test_layout.py
Line 19 should work -###  it is critical. If you debug and have error on this line - check previous errors. If still wrong - good luck.

tldr instructions:
1) you need clean conda or miniconda for python 3.7 (3.6 is acceptable, but definitely not 3.8 - I don't think tmap can be built on this version of python)
2) create an environment so as not to fuck up your own
3) do `conda install conda-build`
4) do `conda build ogdf-conda`
5) do `conda build tmap`
6) install built packages from local source (just google, it is easy)

IMPORTANT -after steps 4 and 5 you can find in console output locations of log files. Save them in safe place just in case.
sidenote - I had pythons 3.6 and 3.7, so two packages were built in my case.
